src/components/assets/EvmAssetList.vue

```diff
@@ -249,24 +213,11 @@ export default defineComponent({
       isHideSmallBalances.value = !isHideSmallBalances.value;
     };
 
-    const handleModalTransfer = ({
-      currency,
-      isOpen,
-    }: {
-      isOpen: boolean;
-      currency: SelectedToken | Erc20Token | string;
-    }): void => {
-      token.value = currency;
-      isModalTransfer.value = isOpen;
-      if (!isOpen) {
-        symbol.value = '';
-      } else if (currency === nativeTokenSymbol.value) {
-        symbol.value = nativeTokenSymbol.value;
-      } else {
-        const c = currency as SelectedToken | Erc20Token;
-        symbol.value = c.symbol;
-      }
-    };
+    const transferLink = computed<string>(() => {
+      const symbol = nativeTokenSymbol.value.toLowerCase();
+      const network = currentNetworkName.value.toLowerCase();
+      return `/assets/transfer?token=${symbol}&network=${network}&mode=local`;
```

Comment:
Another part, since this link has a reference to the network via `network=${network}`, what about moving the network param as part of the base URL. Like `https://portal.astar.network/?rpc=wss%3A%2F%2Fshibuya-rpc.dwellir.com#/assets/transfer/token=${symbol}&mode=local` as an example (like the polkadot js portal).

Reply:
Thank you for your suggestion. 

I've checked [documentation](https://router.vuejs.org/guide/essentials/history-mode.html#hash-mode) but I'm afraid Vue-router doesn't work in that way (adding param before '#'). 

Instead, I can set the ':network' [params](https://router.vuejs.org/guide/essentials/dynamic-matching.html) after /assets, so it looks like `/assets/astar/transfer?token=astr&network=astar&mode=local` Looks good to you?

![image](https://user-images.githubusercontent.com/92044428/187974038-0014971c-49af-4321-a3c2-ecd17783f7e7.png)

Reply:
Hmm, it seems like the string parser does not work well with vue router according to https://github.com/vuejs/vue-router/issues/1259 so the nested query won't work.
Adding a custom parser is too much work too. I like your idea, but instead of putting the network param after the assets, how about moving it to the root (before assets)?

Reply:
I've updated the router logic in [this](https://github.com/AstarNetwork/astar-apps/pull/479/commits/09f01d11f8b9a7892d85517e1d40532136f4430f) commit.
So now, the app network can be changed by typing manually like this:

https://astar-apps--pr479-feat-transfer-page-ka2ln004.web.app/#/astar/assets
https://astar-apps--pr479-feat-transfer-page-ka2ln004.web.app/#/shiden/assets
https://astar-apps--pr479-feat-transfer-page-ka2ln004.web.app/#/shibuya-testnet/assets

Reply:
This is cool!